### PR TITLE
[tiptap] collapsible list add command

### PR DESCRIPTION
**Pull request by danarish4**

### CODE REVIEW

File: packages/ricos-types/web/src/editorCommandsType.ts [hunk 1]
```diff
@@ -100,6 +103,8 @@ export interface PluginsDataMap {
   [FILE_UPLOAD_TYPE]?: any;
   [RICOS_IMAGE_TYPE]?: ImageData;
   [IMAGE_TYPE]?: any;
+  [COLLAPSIBLE_LIST_TYPE]?: any;
+  [RICOS_COLLAPSIBLE_LIST_TYPE]?: any;
```
Comment (maimonav) [inline packages/ricos-types/web/src/editorCommandsType.ts:107]: Shouldn't it be 
[RICOS_COLLAPSIBLE_LIST_TYPE]?: CollapsibleListData; 
?

File: packages/ricos-content/web/src/converters/draft/consts.ts [hunk 2]
```diff
@@ -293,6 +295,8 @@ export const TO_RICOS_PLUGIN_TYPE_MAP = {
   [TEXT_COLOR_TYPE]: RICOS_TEXT_COLOR_TYPE,
   [TEXT_HIGHLIGHT_TYPE]: RICOS_TEXT_HIGHLIGHT_TYPE,
   [VERTICAL_EMBED_TYPE]: RICOS_VERTICAL_EMBED_TYPE,
+  [RICOS_COLLAPSIBLE_LIST_TYPE]: RICOS_COLLAPSIBLE_LIST_TYPE,
```
Comment (maimonav) [inline packages/ricos-content/web/src/converters/draft/consts.ts:298]: Why  `[RICOS_COLLAPSIBLE_LIST_TYPE]: RICOS_COLLAPSIBLE_LIST_TYPE,` is needed?

File: packages/ricos-content/web/src/converters/draft/consts.ts [hunk 3]
```diff
@@ -260,6 +260,8 @@ export const TO_DRAFT_PLUGIN_TYPE_MAP = {
   [LINK_BUTTON_TYPE]: LINK_BUTTON_TYPE,
   [ACTION_BUTTON_TYPE]: ACTION_BUTTON_TYPE,
   [MAP_TYPE]: MAP_TYPE,
+  [RICOS_COLLAPSIBLE_LIST_TYPE]: COLLAPSIBLE_LIST_TYPE,
+  [COLLAPSIBLE_LIST_TYPE]: COLLAPSIBLE_LIST_TYPE,
```
Comment (maimonav) [inline packages/ricos-content/web/src/converters/draft/consts.ts:264]: I'm not sure, but I think you don't even need to add these types to this map, it's for the old Editor Commands usage

File: packages/tiptap-editor/web/src/components/RichContentAdapter/RichContentAdapter.tsx [hunk 4]
```diff
@@ -102,12 +102,13 @@ export class RichContentAdapter implements Omit<TiptapAPI, 'getContent' | 'getCo
         const type = TO_TIPTAP_TYPE[pluginType];
         let id = '';
         if (type) {
+          const { content, ..._attrs } = data;
           id = data.id || generateId();
-          const attrs = { id, ...flatComponentState(data) };
+          const attrs = { id, ...flatComponentState(_attrs) };
```
Comment (maimonav) [inline packages/tiptap-editor/web/src/components/RichContentAdapter/RichContentAdapter.tsx:107]: is this flat usage really needed? AFAIK, you pass only content in data object

Comment (danarish4) [reply packages/tiptap-editor/web/src/components/RichContentAdapter/RichContentAdapter.tsx:107]: needed for loading data (such as loading, tempData...)

File: packages/ricos-content/web/src/converters/draft/consts.ts (hunk 2 above)
Comment (danarish4) [reply packages/ricos-content/web/src/converters/draft/consts.ts:298]: Yep - failed on TS

**Pull request merged**